web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/ListCitizenCertificatesServiceImpl.java

```diff
@@ -61,10 +65,11 @@ public List<CitizenCertificateDTO> get(ListCitizenCertificatesRequest request) {
 
         monitoringLogService.logCertificateListedByCitizen(request.getPersonnummer());
 
-        return certificates
-            .stream()
-            .map(this::getCitizenCertificateDTO)
-            .filter((certificate) -> citizenCertificateFilterService.filter(certificate, request))
+        return Stream.concat(
```

Comment:
Minor thing... maybe concat the stream before monitor logging, just to make sure something doesn't break in fetching from CS and we already logged certificate listed by citizen.